atcoder/maxflow.py

```diff
@@ -0,0 +1,209 @@
+from typing import NamedTuple, Optional, List
+
+
+class MaxFlow:
```

Comment:
@not522 What do you prefer fro the name? Maybe rename to `MFGraph` to match the original impl?